alembic/versions/6e1fe3f1eccf_add_merge_requests_comments_reactions_.py

```diff
@@ -0,0 +1,118 @@
+"""Add merge_requests, comments, reactions tables
+
+Revision ID: 6e1fe3f1eccf
+Revises: 70466f47f936
+Create Date: 2025-04-11 09:52:43.840141
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision: str = "6e1fe3f1eccf"
+down_revision: Union[str, None] = "70466f47f936"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "merge_requests",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column(
+            "mr_id", sa.Integer(), nullable=False, comment="The merge request id"
+        ),
+        sa.Column("project_id", sa.Integer(), nullable=False, comment="The project id"),
+        sa.Column("job_id", sa.Integer(), nullable=False, comment="The job id"),
+        sa.PrimaryKeyConstraint("id"),
+        sa.UniqueConstraint("mr_id", "project_id", "job_id", name="uix_mr_project_job"),
+    )
+    op.create_index(
+        op.f("ix_merge_requests_job_id"), "merge_requests", ["job_id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_merge_requests_mr_id"), "merge_requests", ["mr_id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_merge_requests_project_id"),
+        "merge_requests",
+        ["project_id"],
+        unique=False,
+    )
+    op.create_table(
+        "comments",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column(
+            "merge_request_id",
+            sa.Integer(),
+            nullable=False,
+            comment="The associated merge request id",
+        ),
+        sa.Column("comment_id", sa.Integer(), nullable=False, comment="The comment id"),
+        sa.Column(
+            "created_at",
+            sa.DateTime(),
+            nullable=False,
+            comment="Timestamp when the comment was created",
+        ),
+        sa.ForeignKeyConstraint(
+            ["merge_request_id"],
+            ["merge_requests.id"],
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_index(
+        op.f("ix_comments_comment_id"), "comments", ["comment_id"], unique=False
+    )
+    op.create_index(
+        op.f("ix_comments_merge_request_id"),
+        "comments",
+        ["merge_request_id"],
+        unique=True,
+    )
+    op.create_table(
+        "reactions",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column(
+            "comment_id",
+            sa.Integer(),
+            nullable=False,
+            comment="The associated comment id",
+        ),
+        sa.Column(
```

Comment:
I'm just wondering here; so if a comment has 5 :+1:, does that mean it will have 5 distinct records? if so, we should add something here to distinguish between them, i.e. datetime when the reaction was recorded